.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/tests/eth/ethtests"]
+	path = src/tests/eth/ethtests
+	url = https://github.com/ethereum/tests/
```

Comment:
should we lock a particular commit?

Reply:
Yes absolutely.

Reply:
Yes, locked particular commit, as in reference implementation. For some reason, VMTests was deleted from https://github.com/ethereum/tests/